build-utils/src/index.ts

```diff
@@ -6,6 +6,7 @@ import tar from 'tar';
 import shellEcape from 'any-shell-escape';
 import { spawn } from '@reshuffle/utils-subprocess';
 import { getDependencies, MismatchedPackageAndPackageLockError } from './getdeps';
+import { SpawnOptions } from 'mz/child_process';
```

Comment:
The spawn used here, gets this interface from `child_process`, not `mz/child_process`
please change it for safety